src/utils/utils.cpp

```diff
@@ -601,6 +604,21 @@ void EchoInfo(const std::string &message) {
   }
 }
 
+void EchoExecutionTimeInfo(const std::map<std::string, double> &execution_info) {
+   std::cout << "Additional execution time info:" << std::endl;
+  for (const auto &[key, value] : execution_info) {
+    if (key == "cost_estimate") {
```

Comment:
`cost_estimate` is not a time measure -> `--verbose-query` makes more sense

Reply:
Because of time estimate we are going with `-verbose_query`?